### PR TITLE
Include transformations API in the libs

### PR DESCRIPTION
**Pull request by svix-daniel**
These changes introduce methods for interfacing with the transformations API in the libraries. In the process it updates the `openapi.json` file to the most recent version, meaning some of the unrelated Go generated code is updated by proxy.

### CODE REVIEW

File: rust/src/api.rs [hunk 1]
```diff
@@ -500,6 +501,37 @@ impl<'a> Endpoint<'a> {
...
+        Ok(endpoint_api::get_endpoint_transformation_api_v1_app_app_id_endpoint_endpoint_id_transformation_get(
+			self.cfg,
+			endpoint_api::GetEndpointTransformationApiV1AppAppIdEndpointEndpointIdTransformationGetParams {
+				app_id,
+				endpoint_id,
+				idempotency_key: None,
+			}
+		)
+		.await?)
+    }
```
Comment (svix-andor) [inline rust/src/api.rs:519]: Nit: formatting looks a bit strange here, some lines used tabs it seems?

File: csharp/Svix/Endpoint.cs [hunk 2]
```diff
@@ -633,5 +633,97 @@ public EndpointStats GetStats(string appId, string endpointId, string idempotenc
...
+                return null;
+            }
+        }
+
+        public bool TransformationPartialUpdate(string appId, string endpointId, EndpointTransformationIn endpointTransformationIn, string idempotencyKey = default)
+        {
+            try
+            {
+                var response = _endpointApi.SetEndpointTransformationApiV1AppAppIdEndpointEndpointIdTransformationPatchWithHttpInfo(
+
```
Comment (svix-andor) [inline csharp/Svix/Endpoint.cs:686]: nit: empty line, also in the async variant

File: csharp/Svix/Endpoint.cs [hunk 3]
```diff
@@ -633,5 +633,97 @@ public EndpointStats GetStats(string appId, string endpointId, string idempotenc
...
+        public async Task<bool> TransformationPartialUpdateAsync(string appId, string endpointId, EndpointTransformationIn endpointTransformationIn, string idempotencyKey = default, string cancellationToken = default)
+        {
+            try
+            {
+                var response = await _endpointApi.SetEndpointTransformationApiV1AppAppIdEndpointEndpointIdTransformationPatchWithHttpInfoAsync(
+
+                    appId,
+                    endpointId,
+                    endpointTransformationIn,
+                    idempotencyKey);
```
Comment (svix-andor) [inline csharp/Svix/Endpoint.cs:714]: The cancellation token needs to be passed along here

File: csharp/Svix/Endpoint.cs [hunk 4]
```diff
@@ -633,5 +633,97 @@ public EndpointStats GetStats(string appId, string endpointId, string idempotenc
...
+        }
+
+        public async Task<EndpointTransformationOut> TransformationGetAsync(string appId, string endpointId, string idempotencyKey = default, string cancellationToken = default)
+        {
+            try
+            {
+                var lTransformation = await _endpointApi.GetEndpointTransformationApiV1AppAppIdEndpointEndpointIdTransformationGetAsync(
+                    appId,
+                    endpointId,
+                    idempotencyKey);
```
Comment (svix-andor) [inline csharp/Svix/Endpoint.cs:666]: Cancellation token missing

File: rust/src/api.rs (hunk 1 above)
Comment (svix-daniel) [reply rust/src/api.rs:519]: Weird `cargo fmt` didn't catch that. Fixed :)

**Review by svix-andor — APPROVED** [review]

**Pull request merged**